### ISSUE TITLE
Support title field in configurations

### ISSUE THREAD

**Issue by cgardens** [airbytehq/airbyte#1063]
## Tell us about the problem you're trying to solve
* JsonSchema allows setting a `title` attribute for a field configuration. If the title field is set then the UI should prefer using that field. If it is not set, then it should just use the property name.

```
"basic_normalization": {
  "type": "boolean"
}
```
should display "basic_normalization" as the input label in the ui

```
"basic_normalization": {
  "type": "boolean",
  "title": "Basic Normalization"
}
```
should display "Basic Normalization" as the input label in the ui.

**Comment by Jamakase** [comment]
@cgardens I checked it and as I understand we already support titles.

This is the object I receive from the backend

```
basic_normalization: {
  default: false,
  description: "Whether or not to normalize the data in the destination. See <a href="https://docs.airbyte.io/architecture/basic-normalization.md">basic normalization</a> for more details.",
  examples: [true, false],
  type: "boolean" 
}
```

So I suppose you need to add titles in it.

**Comment by cgardens** [comment]
done. added with this pr: https://github.com/airbytehq/airbyte/pull/1075

**Comment by Jamakase** [comment]
Fixed by #1075
